### PR TITLE
pkp/pkp-lib#9813 upgrade script to fix frong submission progress values inserted by QuickSubmit plugin

### PR DESCRIPTION
**Pull request by bozana**
s. https://github.com/pkp/pkp-lib/issues/9813

### CODE REVIEW

File: dbscripts/xml/upgrade.xml [hunk 1]
```diff
@@ -223,7 +223,11 @@
 	</upgrade>
 
 	<upgrade minversion="3.1.0.0" maxversion="3.4.0.5">
-		<migration class="PKP\migration\upgrade\v3_4_0\I9830_FixEmptyUserLocales" />
+		<migration class="PKP\migration\upgrade\v3_4_0\I9830_FixEmptyUserLocales"/>
+	</upgrade>
+
+	<upgrade minversion="3.4.0.0" maxversion="3.4.0.6">
+		<migration class="PKP\migration\upgrade\v3_4_0\I9813_QuickSubmitSubmissionProgressType"/>
```
Comment (jonasraoni) [inline dbscripts/xml/upgrade.xml:230]: As the current version is `3.4.0-5`, you can move this migration to the tag above.

**Review by jonasraoni — CHANGES_REQUESTED** [review]
Just one fix and I think everything is ok here :)

File: dbscripts/xml/upgrade.xml (hunk 1 above)
Comment (bozana) [reply dbscripts/xml/upgrade.xml:230]: And it does not matter that also 3.1.x.x - 3.3.9.9 would run it too, after the script I7191_SubmissionProgressType has been run :thinking: -- not so clean but...
Or to actually use `<upgrade minversion="3.4.0.0" maxversion="3.4.0.5">`?

Comment (bozana) [reply dbscripts/xml/upgrade.xml:230]: @jonasraoni, OK now?

**Comment by bozana** [comment]
Hi @jonasraoni, I have just moved the upgrade script into the application (OJS) because the QuickSubmit is not supported for OMP and OPS in 3.4 release...
When tests pass I will then merge...

**Review by jonasraoni — APPROVED** [review]
That makes sense :)

**Pull request merged**